lightning/src/ln/functional_test_utils.rs

```diff
@@ -751,6 +751,38 @@ pub fn update_nodes_with_chan_announce<'a, 'b, 'c, 'd>(nodes: &'a Vec<Node<'b, '
 	}
 }
 
+pub fn create_private_chan_between_nodes_with_value<'a, 'b, 'c, 'd>(nodes: &'a Vec<Node<'b, 'c, 'd>>, a: usize, b: usize, channel_value: u64, push_msat: u64, a_flags: InitFeatures, b_flags: InitFeatures) -> (msgs::ChannelUpdate, msgs::ChannelUpdate, [u8; 32], Transaction) {
+	if nodes[a].node.get_current_default_configuration().channel_options.announced_channel {
+		panic!("`Nodes[a]` must have channel_options.announced_channel set to false")
+	}
+	nodes[a].node.create_channel(nodes[b].node.get_our_node_id(), channel_value, push_msat, 42, None).unwrap();
```

Comment:
The last parameter to `create_channel` can be used to override the configuration values, I believe. So I think this function can simply override `channel_options.announced_channel` and re-use the other helpers instead of writing everything below this.

Reply:
Thanks! Ok, I'll look into which other helpers I can use that doesn't require announcement signatures.